openvpn-wire-protocol.xml

```diff
@@ -1462,7 +1462,8 @@ struct key_exchange {
             <li>bit 7: The client is capable of sending exit notification via control channel using <xref target="exitcc">EXIT</xref> message. The client is accpting the protocol-flags pushed option for the EKM capability</li>
             <li>bit 8: The client is capable of accepting <xref target="authfailed">AUTH_FAILED,TEMP</xref> messages.</li>
             <li>bit 9: The client is capable of dynamic tls-crypt</li>
-            <li>bit 10: The client is capable of AEAD tag at the end of a data channel packet and capable of using 64bit packet counters for AEAD ciphers.</li>
+            <li>bit 10: The client is capable of using the epoch data format. This format uses the AEAD tag at the end
```

Comment:
Looks like `and` is missing? Alternatively, `AEAD tag at the end, in addition to 48 bit ... `

Reply:
Yes. will fix that.